examples/asr/emformer_rnnt/pipeline_demo.py

```diff
@@ -1,107 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
let's not touch the examples folder for now. It would require more work. For example this pipeline_demo.py file is mentioned in the README, so removing it would mean updating the readme. We'll do that as follow-ups